### PR TITLE
Docker issue.

### PR DESCRIPTION
**Pull request by jakeharding**
Addresses #46. Disables the volume mounting into the container for now. Removes migrations used by unmanaged models, adds some documentation and a bugfix in the repo details component.

### CODE REVIEW

**Comment by bparish628** [comment]
If you have tried this after `rm -r dist` and no docker images/containers running, then I think this looks find.

File: Dockerfile [hunk 1]
```diff
@@ -6,12 +6,9 @@
  RUN apt-get update && apt-get install -y curl netcat
  RUN curl -sL https://deb.nodesource.com/setup_7.x | bash -
  RUN apt-get install -y nodejs
- RUN mkdir /www
```
Comment (bparish628) [inline Dockerfile:9]: Why are you removing this? I like being explicit in whats happening :smile:

**Review by bparish628 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (jakeharding) [reply Dockerfile:9]: WORKDIR creates the directory of it doesn't exist.

**Review by bparish628 — APPROVED** [review]

**Comment by jakeharding** [comment]
This was tested on a fresh clone of the repo so a dist directory had not been created.

**Pull request merged**